vars/simpleNode.groovy

```diff
@@ -54,3 +53,22 @@ def call(String osName, version, Closure body) {
         }
     }
 }
+
+// Example:
+//
+// simpleNode('OSX10.12', 'latest') { <= braces define the closure, implicitly passed as the last parameter
+//     checkout scm
+//     sh 'echo Hello world'
+// }
+
+// Runs a set of functionality on the default node
+// that supports docker.
+// Parameters:
+//  osName - Docker image to use
+//  imageVersion - Version of the OS image.  See Agents.getMachineAffinity
+//  body - Closure, see example
+def call(String osName, version, Closure body) {
+    simpleNode(Agents.getAgentLabel(osName, version)) {
```

Comment:
In this case you want to pass the body as a parameter:

simpleNode(Agents.getAgentLabel(osName, version), body)

Reply:
Ah ok.

Reply:
Fixed